groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -480,6 +480,26 @@ groups:
       - dlipovetsky@d2iq.com
       - neolit123@gmail.com
 
+  - email-id: sig-cluster-lifecycle-kubespray@kubernetes.io
+    name: sig-cluster-lifecycle-kubespray
+    description: |-
+      Group for Kubespray maintainers to receive CI system notifications.
```

Comment:
but this is not what you had in the pr description?

there are different settings for groups that will only receive notifications and public groups that are for user conversations.

what is the actual purpose of this ML?

cc @VannTen @tico88612

Reply:
The intent was for a private mailing list for the maintainers. What prompted that was missing a notification from the infra provider of our CI (Oracle Cloud Infrastructure) with a need for action, leading to unavailability of our CI cluster. (The notification was sent to a mail which appears to be cncf, probably the organization contact for project using CNCF credits ? Not sure). The notif system is configurable from their console.

I'm not sure how sensible we should consider the notifs, but we erred on the side of caution.

Reply:
ok, looks like i misunderstand and the OP did speak about notifications. sorry about that @yankay 
in that case:

- please use exactly the same settings like
https://github.com/kubernetes/k8s.io/blob/b907fef8009b5b663fe76f57c2624a192d60bf6f/groups/sig-cluster-lifecycle/groups.yaml#L103
- name the ML sig-cluster-lifecycle-kubespray-alerts
- add your owners